reasoner/controller/ControllerRegistry.java

```diff
@@ -153,6 +153,7 @@ public void terminate(Throwable e) {
 
     public void createRootConjunction(ResolvableConjunction conjunction, Set<Variable.Retrievable> filter,
                                       boolean explain, ReasonerConsumer<ConceptMap> reasonerConsumer) {
+        planner().plan(conjunction, new HashSet<>());
```

Comment:
hmm okay so it's odd to do planning in the registry... the registry should only deal with registering actors & managing their lifetimes, and not know/care about planning - that's too low-level.
So - can the root controllers themselves create the plans?